[2021-03-04 20:28:10,747] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: clean_data.load_data 2021-03-04T20:28:04.577958+00:00 [queued]>
[2021-03-04 20:28:10,773] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: clean_data.load_data 2021-03-04T20:28:04.577958+00:00 [queued]>
[2021-03-04 20:28:10,773] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2021-03-04 20:28:10,774] {taskinstance.py:881} INFO - Starting attempt 1 of 2
[2021-03-04 20:28:10,774] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2021-03-04 20:28:10,805] {taskinstance.py:901} INFO - Executing <Task(PythonOperator): load_data> on 2021-03-04T20:28:04.577958+00:00
[2021-03-04 20:28:10,808] {standard_task_runner.py:54} INFO - Started process 4908 to run task
[2021-03-04 20:28:10,836] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', 'clean_data', 'load_data', '2021-03-04T20:28:04.577958+00:00', '--job_id', '41', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/data_clean.py', '--cfg_path', '/tmp/tmpd2nmt0cx']
[2021-03-04 20:28:10,840] {standard_task_runner.py:78} INFO - Job 41: Subtask load_data
[2021-03-04 20:28:10,894] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: clean_data.load_data 2021-03-04T20:28:04.577958+00:00 [running]> 6623e80cd770
[2021-03-04 20:28:10,960] {logging_mixin.py:112} WARNING - /home/airflow/.local/lib/python3.6/site-packages/pandas/core/generic.py:2612: UserWarning: The spaces in these column names will not be changed. In pandas versions < 0.14, spaces were converted to underscores.
  method=method,
[2021-03-04 20:28:11,284] {python_operator.py:114} INFO - Done. Returned value was: None
[2021-03-04 20:28:11,296] {taskinstance.py:1070} INFO - Marking task as SUCCESS.dag_id=clean_data, task_id=load_data, execution_date=20210304T202804, start_date=20210304T202810, end_date=20210304T202811
[2021-03-04 20:28:15,732] {local_task_job.py:102} INFO - Task exited with return code 0
